charts/tigera-operator/README.md

```diff
@@ -146,4 +146,7 @@ tigeraOperator:
   registry: quay.io
 calicoctl:
   image: docker.io/calico/ctl
+
+# Some defaults used in the templates.
```

Comment:
I think this comment should explain how the imagePullPolicy is used. e.g.,

> Configures the image pull policy for all containers in the tigera/operator deployment.

Ideally, this is also used to configure the image pull policy of containers deployed by the operator but it doesn't expose that capability today IIRC.

Reply:
Modified